db/init.sh

```diff
@@ -1,26 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
It may be worth exporting this script as a package so we can consume it more easily on the server. This would mean the script gets built with psql and want dependencies included.

`writeShellApplication` would be the tool of choice for this.

Reply:
Yes and it will be since we'll set .follows for the flake input.